packages/playwright/src/reporters/junit.ts

```diff
@@ -190,7 +198,15 @@ class JUnitReporter extends EmptyReporter {
       for (const attachment of result.attachments) {
         if (!attachment.path)
           continue;
-        const attachmentPath = path.relative(this.config.rootDir, attachment.path);
+
+        let attachmentPath = path.relative(this.config.rootDir, attachment.path);
+        try {
+          const outputFile = this.resolveOutputFile();
```

Comment:
Resolving the output path inside the for loop here seems wasteful. I wonder if we should change the reporter to resolve the output path once at the beginning of the process instead of resolving multiple times.

Reply:
Done